filebeat/examples/security/values.yaml

```diff
@@ -1,12 +1,15 @@
 filebeatConfig:
   filebeat.yml: |
     filebeat.inputs:
-    - type: docker
-      containers.ids:
-      - '*'
+    - type: container
+      paths:
+        - /var/log/containers/*.log
```

Comment:
Just to be sure, is this path actually where hosts' `var/log` is mounted? This is what I expect to see https://github.com/elastic/beats/blob/1a029e547fb1018668e7ab961bcbbe496696ff25/deploy/kubernetes/filebeat-kubernetes.yaml#L110.

Reply:
Thanks for the notice, fixed by https://github.com/elastic/helm-charts/pull/568/commits/72f704ed43f0a04a84cc778ece51acdb83a28b9e